src/App.tsx

```diff
@@ -1,18 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+
+const total = 42;
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
+const items = getNumbers(1, total)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [perPage, setPerPage] = useState<number>(5);
+
+  const START_ITEM = perPage * currentPage - perPage + 1;
+  const END_ITEM = perPage * currentPage > items.length
```

Comment:
better to use uppercase naming for constants only
```suggestion
  const startItem = perPage * currentPage - perPage + 1;
  const endItem = perPage * currentPage > items.length
```